### PR TITLE
Update opencv README

### PR DESCRIPTION
**Pull request by msteffen**

### CODE REVIEW

File: doc/examples/opencv/README.md [hunk 1]
```diff
@@ -108,7 +122,7 @@ You can view this data by mounting it locally:
 ```sh
 $ mkdir /tmp/pfs
 $ pachctl mount /tmp/pfs
```
Comment (JoeyZwicker) [inline doc/examples/opencv/README.md]: Just add `&` to the actual command we list here so usees can copy/paste. 

On a separate note, do we want to them to view the images with FUSE at all? I think opening up the local files in a browser is actually better.

Comment (JoeyZwicker) [inline doc/examples/opencv/README.md]: On nvm. We do have them use the browser

**Comment by JoeyZwicker** [comment]
LGTM

File: doc/examples/opencv/README.md (hunk 1 above)
Comment (msteffen) [inline doc/examples/opencv/README.md]: "&" is added

**Pull request merged**